vim/autoload/himalaya/msg.vim

```diff
@@ -305,6 +305,37 @@ function! himalaya#msg#attachments()
   endtry
 endfunction
 
+function! himalaya#msg#complete_contact(findstart, base)
+  try
+    if a:findstart
+      if !exists("g:himalaya_complete_contact_cmd")
+        echoerr "You must set 'g:himalaya_complete_contact_cmd' to complete contacts"
+        return -3
+      endif
+
+      " search for everything up to the last colon or comma
+      let l:line_to_cursor = getline(".")[:col(".") - 1]
```

Comment:
Could you remove all `l:`? It is just to stay consistent with the existing code base :pray:

Reply:
Done!